packages/catalog-search/src/LearningTypeRadioFacet.jsx

```diff
@@ -12,7 +12,7 @@ import {
   LEARNING_TYPE_COURSE, LEARNING_TYPE_PROGRAM, LEARNING_TYPE_PATHWAY, LEARNING_TYPE_VIDEO,
 } from './data/constants';
 
-const LearningTypeRadioFacet = ({ enablePathways }) => {
+const LearningTypeRadioFacet = ({ enablePathways, enableVideos }) => {
   const { refinements, dispatch } = useContext(SearchContext);
```

Comment:
[suggestion] I see this is largely following parity with the `enablePathways` approach already implemented here. I might suggest an alternative approach to consider that would eliminate the need to prop drill `enableVideos` throughout all the sub-components (e.g., `LearningTypeRadioFacet`, `SearchFilters`, and `SearchHeader`).

You might consider adding the `enableVideos` prop to the `SearchData` component, and passing it through the `searchContext`. That way, the sub-components that rely on `enableVideos` could pull directly from `useContext(SearchContext)` instead of relying on passing the value as a prop throughout.